### PR TITLE
Complete website

### PR DESCRIPTION
**Pull request by Gh-Reza**
## Changes:

- Added header, contact, and footer sections
- Styled the mentioned sections
- Added functionality to the nav links

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,25 +8,85 @@
...
+          </ul>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <p>
+        2023 &copy Copyright
+      </p>
+    </footer>
```
Comment (descholar-ceo) [inline index.html:90]: - [ ] The copyright footer should be touching the bottom edge, yours is a bit far from the bottom edge

### Your design 
Screenshot 2023-06-01 at 09 46 14


### The template
Screenshot 2023-06-01 at 09 17 23

File: index.html [hunk 2]
```diff
@@ -8,25 +8,85 @@
...
+            </li>
 
+            <li>
+              <a href="#" class="nav-link">Add new</a>
+            </li>
+            
+            <li>
+              <a href="#" class="nav-link">Contact</a>
+            </li>
+          </ul>
```
Comment (descholar-ceo) [inline index.html]: - [ ] When clicking a menu, the menu should look activated as the template

### Your design
Screenshot 2023-06-01 at 09 15 12


### Templates
Screenshot 2023-06-01 at 09 11 20

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @Gh-Reza && @hajnaloltyan 🤝 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
👍🏽 PR looks professional with the description and professional commit message ✔️ 
👍🏽 Template is almost followed except for one or two things ✔️ 
👍🏽 Both contributors have contributed ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [x] Your README file has some parts to improve:
    - [ ] On this line: [`git clone git@github.com:Gh-Reza/HelloMicroverse.git`](https://github.com/Gh-Reza/AwsomeBooks/blob/f84fd2402173e2800514a37ab645094bf60000c2/README.md?plain=1#L105) 👈🏽  Would you change this link add this one: `https://github.com/Gh-Reza/AwsomeBooks` 
    - [ ] This Line: [`cd Portfolio-mobile-version`](https://github.com/Gh-Reza/AwsomeBooks/blob/f84fd2402173e2800514a37ab645094bf60000c2/README.md?plain=1#L132) 👈🏽  Replace it with `cd AwsomeBooks` 


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Gh-Reza, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**